crates/rolldown/tests/esbuild/loader/loader_data_url_escape_percents/artifacts.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/loader/loader_data_url_escape_percents
+---
+# Assets
+
+## entry_js.mjs
+
+```js
+
+//#region percents.txt
+var percents_default = "data:text/plain;charset=utf-8,%, %3, %2533, %25333%0A%, %e, %25ee, %25eee%0A%, %E, %25EE, %25EEE";
```

Comment:
Esbuild:
```js
var percents_default = "data:text/plain;charset=utf-8,%0A%, %3, %2533, %25333%0A%, %e, %25ee, %25eee%0A%, %E, %25EE, %25EEE%0A";
```

Reply:
It's because the dataset, esbuild's includes new lines: [here](https://github.com/evanw/esbuild/blob/main/internal/bundler_tests/bundler_loader_test.go#L1099).

Reply:
I dont' mind you add the newlines

Reply:
It's Ok now.